src/admin/components/forms/RenderFieldDescription/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Props } from './types';
+import './index.scss';
+
+const RenderFieldDescription: React.FC<Props> = (props) => {
+  const {
+    description,
+    value,
+    CustomComponent,
+  } = props;
+
+  if (CustomComponent) {
```

Comment:
Here I am prioritizing a CustomComponent over `admin.description` from the config.